ledger/src/blockstore_db.rs

```diff
@@ -1138,6 +1148,15 @@ where
     pub fn put_bytes(&self, key: C::Index, value: &[u8]) -> Result<()> {
         self.backend.put_cf(self.handle(), &C::key(key), value)
     }
+
+    /// Retrieves the specified RocksDB integer property of the current
+    /// column family.
+    ///
+    /// Full list of properties that return int values could be find
```

Comment:
nit: "could be find" ==> "can be found"

Reply:
I was copying the comment from the rust-rocksb API.  Looks like I can earn one PR there :p.

Reply:
Fixed